content/docs/esc/reference/well-known-properties/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Well-Known Properties
```

Comment:
I know that it's more technically correct, but I don't think "well-known" communicates the concept to a practitioner as well as "reserved" (which, if it's technically incorrect, I don't think we should use) or "special" (not very technical, but easy to conceptualize).

Reply:
Hm. On its face "reserved" didn't feel quite right, but the more I consider it the more appropriate it seems. In particular, we do intend to add schema validation for these properties in the future so that we can issue errors from the evaluator if their values are not well-formed. So I think `reserved` feels okay.

Reply:
Done